.travis.yml

```diff
@@ -5,13 +5,16 @@ services:
 
 language: java
 before_install:
+  - docker run --name dbfitdb2 -d -p 50000:50000 -v "`pwd`/test_vm/scripts/db2:/var/dbfit/test_vm/scripts/db2" -e DB2INST1_PASSWORD=db2inst1-pwd -e LICENSE=accept ibmcom/db2express-c:latest db2start &
```

Comment:
I was trying to avoid too many changes to the shell scripts to maintain compatibility when they're being used to install DB2 via vagrant. I'll have another look at the scripts.